### PR TITLE
update ui

### PR DESCRIPTION
**Pull request by barkhabhojak**

### CODE REVIEW

File: app/Widgets/Intranet/Folder.js [hunk 1]
```diff
@@ -50,41 +64,46 @@ export default class Folder extends React.Component {
...
+            timeStamp={this.props.timeStamp}
+            folders={this.props.location.count()}
+            setSearch={this.props.setSearch}
+          />
+        );
     return (
-        <div style={style.main} id="scroller">
-            <Scrollbars style={{ height: window.innerHeight - 50 }}>
-              {isDashboard}
-              {this.displayStructure(this.props.search)}
```
Comment (kepta) [inline app/Widgets/Intranet/Folder.js:84]: why did you remove search?

**Comment by barkhabhojak** [comment]
Damn. Didn't notice. Sorry

**Comment by kepta** [comment]
and also when you open intranet full screen it is still in a box, did you see that ?

File: app/Widgets/Intranet/Folder.js [hunk 2]
```diff
@@ -50,41 +64,46 @@ export default class Folder extends React.Component {
...
+        (
+          <Link to={'intranet'} style={style.head}>
+            Intranet
+          </Link>
+        )
+        :
+        (
+          <Toolbar
+            pathString={this.props.pathString}
+            goToStringPath={this.props.goToStringPath}
```
Comment (kepta) [inline app/Widgets/Intranet/Folder.js:88]: it is goToPath, not goToStringPath.

**Comment by barkhabhojak** [comment]
I'll fix abhi

File: app/Widgets/Intranet/Folder.js [hunk 3]
```diff
@@ -50,41 +64,46 @@ export default class Folder extends React.Component {
...
+            timeStamp={this.props.timeStamp}
+            folders={this.props.location.count()}
+            setSearch={this.props.setSearch}
+          />
+        );
     return (
-        <div style={style.main} id="scroller">
-            <Scrollbars style={{ height: window.innerHeight - 50 }}>
```
Comment (kepta) [inline app/Widgets/Intranet/Folder.js:82]: This is critical for displaying intranet correctly. 
I would recommend refactoring the code, 
and do the thing which you did only when this.props.dashboard is true.
